tests/queries/0_stateless/02515_aggregate_functions_statistics.reference

```diff
@@ -0,0 +1,24 @@
+[[nan]]
```

Comment:
Why are all test results `nan`?

Reply:
Because `corr` use a unstable algorithm, you could have a look at `corr`'s test query in `00181_aggregate_functions_statistics.sql` and `00181_aggregate_functions_statistics.reference`.

Reply:
done.